invenio_users_resources/services/groups/tasks.py

```diff
@@ -18,24 +18,22 @@
 
 
 @shared_task(ignore_result=True)
-def reindex_group(role_id):
+def reindex_groups(role_ids):
     """Reindex the given user."""
     index = current_groups_service.record_cls.index
     if current_groups_service.indexer.exists(index):
         try:
-            group_agg = GroupAggregate.get_record(role_id)
-            current_groups_service.indexer.index(group_agg)
+            current_groups_service.indexer.bulk_index(role_ids)
```

Comment:
Curiosity: we use the group service but we reindex roles??? Maybe it is worth renaming the `role_ids` to `group_ids`?